cli/mk_data/src/generate.rs

```diff
@@ -400,6 +400,10 @@ pub fn run_cmd(
             cmd.env(key, value);
         }
     }
+
+    // Don't want to accidentally merge response files
+    let _ = std::fs::remove_file(output_file);
+
```

Comment:
nonblocking: is this redundant?

Reply:
Yeah I discovered this first, fixed it in the client, then forgot to remove this.